src/Controllers/CheckoutController.php

```diff
@@ -261,31 +283,42 @@ public function doAuth(
 
     /**
      * @param PaymentService $paymentService
-     * @param $ordeId
-     *
+     * @param int $orderId
      * @return string
+     * @throws \Throwable
      */
     public function doAuthFromOrder(
         PaymentService $paymentService,
-        $orderId
-    ) {
-
-        /** @var OrderRepositoryContract $orderContract */
-        $orderContract = pluginApp(OrderRepositoryContract::class);
-
-        /** @var \Plenty\Modules\Authorization\Services\AuthHelper $authHelper */
-        $authHelper = pluginApp(AuthHelper::class);
+        int            $orderId
```

Comment:
Are you sure that in no place NULL or something other than INT is given in here?